.github/workflows/ci.yml

```diff
@@ -23,8 +23,10 @@ jobs:
       - name: Install
         run: |
           npm i -g npm
-          npm ci --prefer-offline --omit=optional
```

Comment:
there seems to be a bug with vitest and this flag.

[Thread for reference](https://github.com/vitejs/vite/discussions/15532)

Without removing this, the CI would run into the following error on the `test` step:

`Cannot find module @rollup/rollup-linux-x64-gnu`